draft blog for may meetup (#233)
nairobilug/nairobilug.or.ke: content/meetup-may-2026-uhuru-uhuru-uhuru-uhuru-tidesdb.md

### Uhuru, Uhuru, Uhuru, Uhuru and MPD as a music streaming service

Benson Muite started his session by speaking about free software as defined by the [Free Software Foundation](https://www.fsf.org/). He addressed the four essential freedoms of free software, namely the freedom to [run, edit, contribute to, and share software](https://www.gnu.org/philosophy/free-sw.html). The freedoms are listed below:

- The freedom to run the program as you wish, for any purpose (freedom 0 / uhuru 0). 
- The freedom to study how the program works, and change it so it does your computing as you wish (freedom 1 / uhuru 1). Access to the source code is a precondition for this.
- The freedom to redistribute copies so you can help others (freedom 2 / uhuru 2).
- The freedom to distribute copies of your modified versions to others (freedom 3 / uhuru 3). By doing this you can give the whole community a chance to benefit from your changes. Access to the source code is a precondition for this.

> 

Reviewer: ```suggestion

Participants shared free applications that they regularly use.  They also discussed restrictions to freedom on some cloud software as a service platforms.

```

Author: Thank you for this.

> Benson then took us through how we could set up our own streaming service using [Music Player Daemon (MPD)](https://www.musicpd.org/). MPD is a powerful application that runs on Linux. It can play a wide variety of sound files while being controlled through the network. 

Reviewer: ```suggestion
Benson then did a demo of using [Music Player Daemon (MPD)](https://www.musicpd.org/) as a local streaming service. MPD is a powerful application that runs on Linux and Android. It can play a wide variety of sound files while being controlled through the network. Attendees tried to control MPD using [M.A.L.P.](https://f-droid.org/en/packages/org.gateshipone.malp/) from their phones, but only a few could control and hear sound simultaneously with the setup provided.
```

Watch Grivine's talk below. 

https://youtu.be/xb9fsBoWpEs?si=9-wswZqn1GSGXj9n

Subscribe and comment!

### Food, networking and stuff

The attendees brought snacks and we shared it amongst ourselves. Check out the photos below.

![May-meetup-group-photo.jpg]({static}/images/meetup-may-2026/may-meetup-group-photo.jpg "May meetup group photo"){: .image-process-large-photo}

### Acknowledgements

I thank Benson, Gravine and Daniel for taking the time to prepare for their sessions.

Shout-out to Douglas for always delivering on the posters. I'm already looking forward to the next one.

> Shout-out to Hesus for taking the picturs and videos. Your services are always appreciated.

Reviewer: ```suggestion
Shout-out to Hesus for taking the pictures and videos. Your services are always appreciated.
```
